vm/instruction.go

```diff
@@ -453,6 +453,8 @@ func (vm *VM) initObjectFromGoType(value interface{}) Object {
 	switch v := value.(type) {
 	case int:
 		return vm.initIntegerObject(int(v))
+	case int64:
```

Comment:
But for now, the object mapper deserves another issue or PR to discuss about it @@